src/libNode/Node.cpp

```diff
@@ -1069,26 +1070,32 @@ bool Node::ProcessTxnPacketFromLookup(
            0) ||
           m_justDidFallback))) {
       lock_guard<mutex> g2(m_mutexTxnPacketBuffer);
-      m_txnPacketBuffer.emplace(epochNumber, message);
+      m_txnPacketBuffer.emplace_back(message);
       return true;
     }
   }
 
-  if (epochNumber < m_mediator.m_currentEpochNum) {
-    LOG_GENERAL(WARNING, "Txn packet from older epoch, discard");
-    return false;
-  } else if (epochNumber == m_mediator.m_currentEpochNum) {
-    return ProcessTxnPacketFromLookupCore(message, shardId, transactions);
-  } else {
+  //
+
+  if (m_mediator.m_lookup->IsLookupNode(from)) {
```

Comment:
Can you add some description here about why we only buffer if it's from lookup, and not call `ProcessTxnPacketFromLookupCore`?